base/src/main/AndroidManifest.xml

```diff
@@ -23,12 +25,17 @@
         android:icon="@mipmap/ic_launcher"
         android:label="@string/app_name"
         android:supportsRtl="true"
-        android:theme="@android:style/Theme.DeviceDefault">
+        android:theme="@android:style/Theme.DeviceDefault"
+        tools:ignore="AllowBackup">
 
+        <meta-data
+            android:name="com.google.android.wearable.standalone"
+            android:value="true" />
         <service
-            android:name=".CustomComplicationProviderService"
+            android:name="com.example.android.wearable.complicationsdatasource.CustomComplicationDataSourceService"
+            android:exported="true"
             android:icon="@drawable/ic_watch_white"
-            android:label="@string/complications_provider_service_label"
+            android:label="@string/complications_data_sources_service_label"
             android:permission="com.google.android.wearable.permission.BIND_COMPLICATION_PROVIDER">
```

Comment:
No action required, just acknowledging that this permission doesn't change from `PROVIDER` to `DATA_SOURCE`.

Reply:
I wonder if I should call that out in the comments.

Hmm... should we log a bug for this? Seems odd.

Reply:
I think this is still required, since the underlying permission hasn't changed. But it'd definitely be good to confirm and especially call it out on the revamped version of https://developer.android.com/training/wearables/watch-faces/exposing-data-complications